config.json

```diff
@@ -445,6 +445,29 @@
           "structs"
         ],
         "status": "beta"
+      },
+      {
+        "slug":"weather-forecast-server",
+        "name":"Weather Forecast Server",
+        "uuid":"de69aae1-95b0-4ce2-870b-db63bb71c11a",
+        "concepts": [
+          "http-server"
+        ],
+        "prerequisites":[
+          "interfaces",
+          "structs"
+        ]
+      },
+      {
+        "slug":"weather-forecast-client",
+        "name":"Weather Forecast Client",
```

Comment:
Since this a some sort of follow up for the main Weather Forecast exercise, do you think it is consistent to do so? I'm ok with the change personally, I am only worried if this might be easier to make sense of based on the story.

Reply:
Since the weather forecast exercise is only about comments and is not big on the story element, the "follow up" relationship is rather weak imo. I would go with the the shorter version but its ok as is as well.